polars/polars-lazy/polars-plan/Cargo.toml

```
@@ -17,7 +17,7 @@ polars-io = { version = "0.26.1", path = "../../polars-io", features = ["lazy",
 polars-ops = { version = "0.26.1", path = "../../polars-ops", default-features = false }
 polars-time = { version = "0.26.1", path = "../../polars-time", optional = true }
 polars-utils = { version = "0.26.1", path = "../../polars-utils" }
-pyo3 = { version = "0.16", optional = true }
+pyo3 = { version = "0.18", optional = true }
```

Similarly: must this include the bug-fix number? I.e. must it be:
```suggestion
pyo3 = { version = "0.18.0", optional = true }
```
?